components/base/kinematicbase/ptgKinematics.go

```diff
@@ -0,0 +1,122 @@
+// Package kinematicbase contains wrappers that augment bases with information needed for higher level
+// control over the base
+package kinematicbase
+
+import (
+	"context"
+	"errors"
+	"math"
+	"time"
+
+	"github.com/golang/geo/r3"
+	"go.uber.org/multierr"
+	utils "go.viam.com/utils"
+
+	"go.viam.com/rdk/components/base"
+	"go.viam.com/rdk/motionplan/tpspace"
+	"go.viam.com/rdk/referenceframe"
+	rdkutils "go.viam.com/rdk/utils"
+)
+
+// Define a default speed to target for the base in the case where one is not provided.
+const defaultBaseMps = 0.6
```

Comment:
this should be in mm per sec to follow unit conventions in other parts of the kinematicbase package.

Reply:
wait wait, but PTGs expect meters right?

Reply:
Everything has been updated to expect mm